lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -24,7 +24,7 @@ class MarriageAbroadDataQuery
 
     OS_CONSULAR_CNI_IN_NEARBY_COUNTRY = %w(nicaragua)
 
-    OS_AFFIRMATION_COUNTRIES = %w(belgium cambodia colombia ecuador egypt lebanon finland macao mongolia morocco norway peru qatar south-korea united-arab-emirates vietnam)
```

Comment:
@cbaines thats a good question.. it appears to state for opposite sex affirmation countries.. OS_AFFIRMATION_COUNTRIES holds a list of these countries. See https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer/calculators/marriage_abroad_calculator.rb#L170